actions.yaml

```diff
@@ -6,3 +6,26 @@ reset-instance:
     Set a new server_name before running this action.
     Once a server_name is configured, you must start a new instance if you wish a different one.
     This actions will erase all data and create a instance with the new server_name.
+register-user:
+  description: |
+    Registers a user for the Synapse server.
+    You need to supply a user name and user password
+    and whether that user should be an admin or not.
+  properties:
+    username:
+      description: |
+        When not using SSO, a user name is needed
+        for the creation of a matrix account.
+      type: string
+    password:
+      description: |
+        Specifies a user password that relates to a user name
```

Comment:
"Password for the user account" seems enough description to me :)

Reply:
Agree :laughing: David suggested that the action could create the password so this field was removed.